space-cli/cmd/modules/database/getters.go

```diff
@@ -22,22 +21,23 @@ func GetDbRule(project, commandName string, params map[string]string) ([]*model.
 	var objs []*model.SpecObject
 	for _, item := range payload.Result {
 		obj := item.(map[string]interface{})
-		for key, value := range obj {
-			str := strings.Split(key, "-")
-			if str[1] == "event_logs" || str[1] == "invocation_logs" {
-				continue
-			}
-			meta := map[string]string{"project": project, "col": str[1], "dbAlias": str[0]}
-
-			delete(obj, "schema")
-
-			// Generating the object
-			s, err := utils.CreateSpecObject("/v1/config/projects/{project}/database/{dbAlias}/collections/{col}/rules", commandName, meta, value)
-			if err != nil {
-				return nil, err
-			}
-			objs = append(objs, s)
+		table := obj["table"].(string)
+		dbAlias := obj["dbAlias"].(string)
+		if table == "event_logs" || table == "invocation_logs" {
+			continue
 		}
+		meta := map[string]string{"project": project, "col": table, "dbAlias": dbAlias}
+
+		delete(obj, "schema")
```

Comment:
I think this is not required, as the dbRule object in gateway doesn't contain schema field. So this line of code is not useful

Reply:
Yes
But @YourTechBud said that it was there previously, and hence do not remove it as it wont create any difference.